### PR TITLE
#60 Translate N.N.N-(rc|alpha|beta).N versions to PEP440

### PR DESCRIPTION
**Pull request by epwilkins**

### CODE REVIEW

File: habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/AbstractHabushuMojo.java [hunk 1]
```diff
@@ -97,12 +101,14 @@ public abstract class AbstractHabushuMojo extends AbstractMojo {
      * property is true, Habushu may override the pyproject.toml defined version in
      * the following build phases/mojos:
      * <ul>
-     * <li>validate ({@link ValidatePyenvAndPoetryMojo}): Automatically sets the
+     * <li>initialize ({@link InitializeHabushuMojo}): Automatically sets the
```
Comment (epwilkins) [inline habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/AbstractHabushuMojo.java:104]: I: This was just documented incorrectly. It happens in the initialize phase.

File: habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/PublishToPyPiRepoMojo.java
Comment (epwilkins) [inline habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/PublishToPyPiRepoMojo.java:1]: I: Just formatting changes

File: habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/exec/PoetryCommandHelper.java
Comment (epwilkins) [inline habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/exec/PoetryCommandHelper.java:1]: I: Just formatting changes

File: habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/exec/ProcessExecutor.java [hunk 2]
```diff
@@ -76,7 +76,6 @@ public String executeAndGetResult(final Logger logger) {
         }
     }
 
-    @SuppressWarnings("deprecation")
```
Comment (epwilkins) [inline habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/exec/ProcessExecutor.java:79]: I: This is the only real change and it was an unnecessary suppression. The rest is just formatting.

File: habushu-maven-plugin/src/test/resources/specifications/package-version-override.feature [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  Scenario Outline: SemVer 2 POM versions are translated to their PEP 440 equivalent
+    Given a Habushu module with a POM version of "<semver>"
+    When the POM version is translated to a PEP 440 version
+    Then the pyproject.toml file is updated with the version "<pep440>"
+
+    Examples:
+      | semver        | pep440       | comment        |
+      | 1.2.3         | 1.2.3        | final release  |
+      | 1.2.3-rc.4    | 1.2.3rc4     | semver 2 rc    |
+      | 1.2.3-rc4     | 1.2.3-rc4    | semver 1 rc    |
```
Comment (epwilkins) [inline habushu-maven-plugin/src/test/resources/specifications/package-version-override.feature:13]: I: SemVer 1.0 is actually compatible with PEP440, so no translation is necessary.

**Review by d-ryan-ashcraft — APPROVED** [review]

**Pull request merged**